package.json

```diff
@@ -3,9 +3,9 @@
   "version": "7.1.0",
   "description": "babel module loader for webpack",
   "files": [
-    "lib"
+    "src"
   ],
-  "main": "lib/index.js",
```

Comment:
Ah I see, because the build script outputs to `/lib`. I mainly made this change so I could install from my git branch for testing and didn't really think about it. 

That may also explain why it isn't working with Node <8.0. I still have to do some testing of this today, but I'll revert that.